assets/src/components/copy-button/index.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { Button } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import './style.scss';
+
+export class CopyButton extends Component {
```

Comment:
Why not a functional component? 🤔

Reply:
There is a `ClipboardButton` component now in the plugin which can be used instead which was copied from Gutenberg in #6320 by @delawski:

https://github.com/ampproject/amp-wp/blob/b795b297ce2d5600782e4622f2e0ff4a8dbff131/assets/src/components/clipboard-button/index.js#L15-L59

It can be used instead of `CopyButton` I think?

Reply:
Yes, We can use `ClipboardButton` instead of this.